src/pages/SupplierInvoicePage.js

```diff
@@ -444,18 +448,20 @@ export class SupplierInvoicePage extends React.Component {
         );
       case FRIDGE_SELECT:
         return (
-          <AutocompleteSelector
-            options={database.objects('Location')}
-            queryString="description BEGINSWITH[c] $0"
-            sortByString="description"
-            onSelect={item => {
+          <GenericChoiceList
+            data={database.objects('Location')}
```

Comment:
data this.FRIDGES ?

Reply:
oops.. 567e0dc82c946f742fae3b149096f4b1a5641672